### PR TITLE
Step 1-2.  Added docs and gitignore.

### PR DESCRIPTION
**Pull request by akimov228aleksei**
Documentation created, gitignore file added.

### CODE REVIEW

File: docs/SRS.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- # External interface requirements
+    + ## User interfaces: 
+       The user will interact with the application through a graphical interface. The user will have access to 3 sections: employees, departments, vacations.
+  
+       Department: This mode is intended for viewing the list of departments, description, creation date, update date.    
+       Employee: The mode is intended for displaying the list of employees, their salaries, department, job assignment and position.    
+       Vacation: This mode is intended for viewing employees on vacation, deleting and editing the vacation date.
+  
+       An exemplary representation of the interface is shown in the figure:
+       ![](https://i.imgur.com/4TH6N0W.png)
```
Comment (jimmyhollywood) [inline docs/SRS.md:49]: Let's upload all images to the repository in `/docs/img` or something like that

File: docs/SRS.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- # Non functional requirements 
+    + ## Software quality attributes
+       To check the quality of the code, use:
+        - Unit tests and integration tests
+        - Coverage  
+        - Linters 
+    
+    + ## Security requirements 
+       For security reasons, some functions are not available for certain categories of users:
```
Comment (jimmyhollywood) [inline docs/SRS.md:64]: I suggest using roles. Admin, moderator or manager and employee.
Admin should have all privileges - read/write.
Moderator/manager should have write permissions - which means read/add/edit/delete.
Employee should only have read permissions.

Note: Any user should not have access to the system until authorization.

File: docs/SRS.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+- # Introduction
```
Comment (jimmyhollywood) [inline docs/SRS.md:1]: Why are you using dashes and plus signs in headers?

There is an example of using of markdown elements:

# Header 1

## Topic 1

Some text

## Topic 2

Enumeration:
- Item 1
- Item 2

# Header 2

etc...

**Review by jimmyhollywood — CHANGES_REQUESTED** [review]
1. Rename PR title according to contained changes
2. Fix comments in review

File: docs/SRS.md (hunk 1 above)
Comment (akimov228aleksei) [reply docs/SRS.md]: Fixed

File: docs/SRS.md (hunk 2 above)
Comment (akimov228aleksei) [reply docs/SRS.md]: Fixed

File: docs/SRS.md (hunk 3 above)
Comment (akimov228aleksei) [reply docs/SRS.md]: Fixed

**Review by jimmyhollywood — APPROVED** [review]

**Pull request merged**